### ISSUE TITLE
Doesn't work on Imgur gifv links?

### ISSUE THREAD

**Issue by rruester** [dadler/thumbnail-zoom#238]
When hovering over a .gifv link, TZP is requesting a .webm instead. Normally Imgur redirects these requests to the gifv image, but for TZP's requests Imgur returns a 404.

**Comment by dadler** [comment]
Yes, it looks like imgur.com changed their site and broke the ability for TZP to show pop-ups for gifv's.  Imgur.com used to provide a corresponding webm when TZP changed .gifv to .webm.  TZP did that since webm was a codec (format) which Firefox can show in a pop-up; gifv isn't.  But now imgur.com doesn't provide webm format anymore; it just redirects to .gifv.

I may have to have it show .gif instead of .gifv or .webm, which would be unfortunate since it'd use more network bandwidth.

**Comment by dadler** [comment]
I will soon prevent TZP from even trying to show pop-ups for gifv's (so at least we won't see an error icon).
